wp-cli/vip-migrations.php

```diff
@@ -70,6 +70,12 @@ function cleanup( $args, $assoc_args ) {
 			}
 		}
 
+		// Cleanup roles
+		if ( get_role( 'administrator' ) === null ) {
```

Comment:
I think this may be an edge case but any concerns with a site that explicitly removes the `administrator` role? One issue would be that we'd then wipe out any custom roles as well. Not sure if this is anything we need to worry about but thought I'd flag it just in case.

Reply:
Yeah, I was thinking about that. The original issue is that `jetpack-start connect` silently fails if the Administrator role doesn't exist (because our machine user is an Administrator).

I guess we could create a custom role for the machine user that's defined in code, so it's always there.

We still need to populate roles after a migration if the option is empty, but maybe that's a separate issue.

Reply:
Do none of the roles exist? Would that mean that `wp_roles()->roles` would return empty (https://developer.wordpress.org/reference/classes/wp_roles/)? Should we just check for that?

Reply:
It returns the vip-support role because it's hard-coded. We could check for that, but again, if someone adds a role in their theme the same way we add vip-support, it will also be there.

Reply:
I believe the odds of having someone removing the `administrator` role are higher than the odds of someone hard-coding roles like we do for `vip-support`, so I'm not sure which one is the best approach, but I think checking for the `administrator` role is the less risky option. 

I'm fully open to suggestions and alternative on how to address this edge case :)

Reply:
> I believe the odds of having someone removing the administrator role are higher than the odds of someone hard-coding roles like we do for vip-support

The problem is that our helper functions hard-code the role and any sites coming from WP.com use those helper functions, so there will certainly be some sites that do this.

What if we unconditionally reset the roles after a migration?